.eslintrc.json

```diff
@@ -21,6 +22,10 @@
         "named": "never",
         "asyncArrow": "always"
       }
-    ]
+    ],
+    "testing-library/await-async-query": "error",
+		"testing-library/no-await-sync-query": "error",
+		"testing-library/no-debugging-utils": "warn",
+		"testing-library/no-dom-import": "off"
   }
```

Comment:
Could you change the indents of the previous three lines from tabs to spaces? The prettier plugin for VSCode should automatically make these spaces.